lib/openshift/cluster_log_forwarder.rb

```diff
@@ -60,6 +60,17 @@ def status_raw(user: nil, quiet: false, cached: true)
         return raw_resource(user: user, cached: cached, quiet: quiet).dig('status')
     end
 
+    def outputs_status(user: nil, quiet: false, cached: false)
```

Comment:
We should default cached to true. Set cached to false in your scenario when you do not want to use cached value.

Reply:
Done, https://github.com/openshift/cucushift/pull/8173 is updated too.